packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -21,7 +23,7 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
       },
       [onChange]
     );
-    const styles = useStyles2(getCheckboxStyles);
+    const styles = useStyles2(getCheckboxStyles(size));
```

Comment:
it's better to not have the styles built every render for differnet props (size), it makes the memoization not work, it's a lot better (more performant) to define the different sizes as different classes instead and pick the right class based on size. 
Normally this does not matter that much but for a small component like this that can exist many many times on a list page it does matter.